docs/Commands.md

```diff
@@ -970,7 +1011,7 @@ Array Reply: All the distinct tags in the tag index.
 
 #### Complexity
 
-O(n), n being the cardinality of the tag field.
+O(n), n being the cardinality of the tag attribute.
```

Comment:
@emmanuelkeller This doesn't make sense to me. How can a field have a cardinality? Is `n` the number of distinct values stored? Why not say it more directly?